woocommerce/woocommerce
# [Launch your store] Add Coming Soon page templates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45113

This PR adds templates for each of the two styles for the Coming Soon page.

<img width="796" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/be89618b-be24-45d1-bb33-1bbf96f2d2b5">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a JN site with `launch-your-store` flag turned on Tools > WCA Test Helper > Features > launch-your-store
2. Make sure Coming Soon mode is on and `Restrict to store pages only` is not toggled on: `/wp-admin/admin.php?page=wc-settings&tab=site-visibility`
3. Visit the Coming Soon page and see how it compares to the design in the screenshot.
4. Change the setting `Restrict to store pages only` toggled on, save changes
5. Revisit the Coming Soon page and see its content and template updated.
6. See the templates and, if you like, edit them: Appearance > Editor > Templates > Coming Soon Entire Store / Coming Soon Store Only



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Expands unreleased feature

</details>
